### DISCUSSION TITLE
快速向模拟器注入大量内存

### DISCUSSION THREAD

**Discussion by Physics365** [Physics365/991CN-X-CW-Decompilation#38] [Q&A]
模拟器能不能快速复制粘贴大量内存？

现在比如放一个 @Xyzstk 的内存编辑器在模拟器里得一个一个字节敲进去，能不能按一下Ctrl+V直接复制或者有别的命令行方法（好像是有的，但是我忘了）

**Comment by bczhc** [comment]
user的那个模拟器有个lua控制台，但CasioEmu(X|Neo)给它去了。用lua控制台的话倒可以用它提供的lua函数实现批量内存写入。其实最好还是期望[imgui_club](https://github.com/ocornut/imgui_club)的内存编辑器能加个范围选中及多字节复制粘贴功能。

**Comment by bczhc** [comment]
我修改了[qiufuyu123/CasioEmuNeo](https://github.com/qiufuyu123/CasioEmuNeo)，支持了多字节粘贴。

https://github.com/bczhc/CasioEmuNeo/tree/multibyte-pasting

**Reply by Physics365** [reply]
支持，等待 @qiufuyu123 合并

**Comment by fish4terrisa-MSDSM** [comment]
我之前试了casioemuneo的从字符串加载rop的功能，不过发现它没办法从字符串中加载字母（F2会变成52之类），不知道是什么问题。。。

**Reply by qiufuyu123** [reply]
小写，不能大小
